src/utils/mtev_security.c

```diff
@@ -135,7 +135,10 @@ mtev_security_usergroup(const char *user, const char *group, mtev_boolean effect
           pwnam_buflen = safe_size * 2;
           safe_size = MAX(grnam_buflen, pwnam_buflen);
           free(mallocd);
-          buf = mallocd = malloc(safe_size);
+          if(safe_size >= 0) 
```

Comment:
Change this to an: `mtevAssert(safe_size > 0);` and remove the else and `printf` -- we can't use printf here as we may not be connected to standard output.